yorkie/src/main/kotlin/dev/yorkie/document/operation/OperationInfo.kt

```diff
@@ -0,0 +1,60 @@
+package dev.yorkie.document.operation
+
+import dev.yorkie.document.crdt.TextWithAttributes
+import dev.yorkie.document.time.TimeTicket
+
+/**
+ * [OperationInfo] represents the information of an operation.
+ * It is used to inform to the user what kind of operation was executed.
+ */
+public sealed class OperationInfo {
+
+    public abstract var path: String
```

Comment:
I defined `path` as a mutable property because making it read-only seemed to result in clumsy boilerplate code for updating its value.
but I’m open to refactoring it if a better solution exists.